File: 0001-two-sum/0001-two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int k) {
        int index[]=new int[2];
        
        for(int i=0;i<nums.length;i++)
        {
            for(int j=i+1;j<=nums.length-1;j++)
          {
                  if(nums[i]+nums[j]==k){
            
                        index[0]=i;
                        index[1]=j;
            
                  }
          }
        
    }
   return index;
    }
}

File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums, k):
        index = [0, 0]
        
        for i in range(len(nums)):
            for j in range(i + 1, len(nums)):
                if nums[i] + nums[j] == k:
                    index[0] = i
                    index[1] = j
                    return index 
        
        return index  
